src/types/CreateTaskOptions.interface.ts

```diff
@@ -28,6 +30,7 @@ export interface CreateTaskOptions {
   web3FunctionHash?: string;
   web3FunctionArgs?: Web3FunctionUserArgs;
   web3FunctionArgsHex?: string;
+  triggerConfig: TriggerConfig;
```

Comment:
I think I would actually prefer:
```suggestion
  trigger: TriggerConfig;
```

the `Config` seems unnecessary

Reply:
fixed 0bf10e5123ee4affee7084cbd7af3d7a16c62f6f